packages/orchestration/src/exos/ica-account-kit.js

```diff
@@ -42,7 +41,9 @@ export const IcaAccountI = M.interface('IcaAccount', {
   reactivate: M.call().returns(VowShape),
 });
 
+// XXX none of these modifiers are working to exclude this type from api-docs
```

Comment:
oh. so it's _not_ yet working?

Reply:
in some cases, no. maybe on types that are transitively included

Reply:
in some cases, no. maybe on types that are transitively included